src/utils/use-transportation-color.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  TransportMode,
+  TransportSubmode,
+} from '@atb/components/transport-mode/types';
+import { useTheme } from '@atb/modules/theme';
+import { TransportColor } from '@atb/modules/theme';
+
+export function useTransportationColor(
```

Comment:
Is this the same as `modeToColor()`?

Reply:
Yes, more or less, and should not be duplicated. I'll update 👍